x/concentrated-liquidity/incentives.go

```diff
@@ -1018,6 +1023,7 @@ func (k Keeper) collectIncentives(ctx sdk.Context, sender sdk.AccAddress, positi
 	}
 
 	// If no incentives were collected, return an empty coin set.
+	// @Dev: Does collectedIncentivesForPosition = 0 => forfeitedIncentivesForPosition = 0?
 	if collectedIncentivesForPosition.IsZero() {
 		return collectedIncentivesForPosition, forfeitedIncentivesForPosition, nil
 	}
```

Comment:
No, I think this check exists because if there are no collected incentives for position, we don't need to continue and do bank sends so we can return all we have early

Reply:
nit: I think event is not being emitted if we return here